attachment_log/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,4 @@
+This module adds the following features for attachments:
```

Comment:
This module shows attachment information in chatter. Following data is displayed:

- User, who has created the attachment
- Attachment creation date

It also allows to log attachment related actions in the record chatter.